### ISSUE TITLE
bug - display username minimum character requirement for registration

### ISSUE THREAD

**Issue by c-nv-s** [makeless/makeless-demo#28]
At the moment there does not seem to be a minimum character warning/error specifically for the username field on the registration page.

on other pages the input field gets outlined red with the warning text " X is required and must be 4 to 50 characters long "

during registration there is no indication that the username must be a minimum character length and if I didn't check the browser dev tools 400 error http response:
```
error: Key: 'Register.Name' Error:Field validation for 'Name' failed on the 'min' tag
```
I would have had no idea why registration was failing.

**Comment by loeffel-io** [comment]
Hey @c-nv-s , 

you are right - we have to update the validations: https://github.com/makeless/makeless-ui/blob/master/src/components/pages/Register.vue#L110

We will take care about this soon - You can also create a PR for this or just override the method by yourself ✌️

**Comment by loeffel-io** [comment]
@loeffelr we also need to update the makeless-ui version

**Comment by c-nv-s** [comment]
I also had a question ( happy to raise it as a separate ticket if you like ).
I was initially going to install and test this from behind a reverse proxy, but it seems that at the moment it is only possible by exposing the backend as a separate port through the proxy as well.
would it not be useful to also allow the backend to be exposed on a separate hostname so that you can have e.g.
frontend - http://account.mysite.com
backend - http://api.mysite.com
this would allow users to keep the entire app accessible on ports 80/443 ?
or have I overlooked something and this is actually possible already ?

**Comment by loeffel-io** [comment]
Hey @c-nv-s, 

not completely sure if we both talk about the same, but you can set your host in the makeless.json file to any host you want: https://github.com/makeless/makeless-demo/blob/master/makeless.json#L5

Also make sure that you update your cors allowed origins: https://github.com/makeless/makeless-demo/blob/master/docker-compose.yml#L39

**Comment by c-nv-s** [comment]
oh ok thanks, I'll try this out later.
I assume the cors origins are comma separated?
```
ORIGINS="http://account.mysite.com,http://api.mysite.com"
```

**Comment by loeffel-io** [comment]
yes 👍

**Comment by loeffelr** [comment]
@c-nv-s validation fixed in v1.0.0-beta.24
Further i will close this ticket.
